src/hub.html

```diff
@@ -241,7 +203,6 @@
         </a-assets>
 
         <!-- Interactables -->
-        <a-entity id="counter" networked-counter="max: 3; ttl: 120"></a-entity>
```

Comment:
We no longer have a TTL, but I think with the delete button + `destroy-at-extreme-distances` this is ok.